modules/nf-core/diamond/blastp/meta.yml

```diff
@@ -22,11 +22,11 @@ input:
   - fasta:
       type: file
       description: Input fasta file containing query sequences
-      pattern: "*.{fa,fasta}"
+      pattern: "*.{fa,fasta,fa.gz,fasta.gz}"
   - db:
```

Comment:
If `meta2` is good, don't forget to add it here

Reply:
Done